grafana/k6-docs
# feat: introduced custom anchors functionality

**Describe the changes this PR brings**
This PR adds the custom md anchors functionality for `h2` and `h3` headers. 

Changes:
- `h2` and `h3` headings syntax now allows to write `{#custom-anchor}` appendix whose content will be used as a replacement for default autogenerated anchor based on heading plain content
- extended `Headings` chapter in [CONTRIBUTING_FILE_FORMAT.md](https://github.com/grafana/k6-docs/blob/custom-md-anchors/CONTRIBUTING_FILE_FORMAT.md#headings-with-custom-anchors)

**Steps to test**
1. Go to the [staging env](https://mdr-ci.staging.k6.io/docs/refs/pull/741/merge/getting-started/installation/)
2. Check out test headings
3. Ensure UI is alright
4. Open up developers tools, explore markup, ensure the `id` at headings and anchor link is generated correctly for every entity
5. Make sure [real-world md syntax](https://github.com/grafana/k6-docs/pull/741/files#diff-9c262732508bb96ca2301a9874b43d27a969e32e44d65bad0ff663e70e48cb93R8) meets the needs 

**TODO**
- [ ] remove test changes in `02 Installation.md` before merging

Closes: #677
